### PR TITLE
feat(auto-tracker): Add auto tracking capability to extension

### PR DESCRIPTION
**Pull request by with-shrey**
Please remember the [Contributing Guidelines](https://github.com/toggl/track-extension/blob/master/docs/CONTRIBUTING.md) :heart:

## :bug: Recommendations for testing

All changes should be tested across Chrome and Firefox.

### CODE REVIEW

**Comment by with-shrey** [comment]
I dont have heap access @izontm 

Please share top 10 integration list so we can validate for them

**Comment by izontm** [comment]
@with-shrey  for the last 7 days
_Sorry, I didn't have time to test it; I can try next week if not tomorrow._
image

File: src/content/notion.js [hunk 1]
```diff
@@ -35,54 +37,63 @@ togglbutton.render(
...
     }
   }
 );
 
-togglbutton.render(
-  '.notion-topbar-action-buttons:not(.toggl)',
-  { observe: true },
-  function (elem) {
-    if (!elem) return;
+setTimeout(() => {
```
Comment (askides) [inline src/content/notion.js:47]: Why the setTimeout here? Does the debounceInterval set to 2000 does not work?

**Review by askides — COMMENTED** [review]
One thing i wanted to be sure about.

Rest is LGTM.

File: src/content/notion.js (hunk 1 above)
Comment (with-shrey) [reply src/content/notion.js:47]: It doesnt do ot for the first render

**Pull request merged**